packages/i18n-calypso/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Trunk
+
+- Breaking: Emitter object replaced with subscriber
+  - Before: `i18n.on( 'change', callback ); i18n.off( 'change', callback );`
+  - After: `const unsubscribe = i18n.subscribe( callback ); unsubscribe();`
+
```

Comment:
Could we keep the old `on` / `off`  APIs around for back-compat (even if deprecated), and have them internally call the new `subscribe` / `unsubscribe` APIs?

If the required efforts are not large, avoiding a breaking change could make it earlier for consumers to update their versions